tests/bootstrap.php

```diff
@@ -34,3 +34,7 @@ class_alias('\PHPUnit_Framework_Error_Warning', '\PHPUnit\Framework\Error\Warnin
     $patchGeneratorPath = __DIR__ . '/bootstrap/PHPUnit_Framework_MockObject_Generator_7.4.php';
     file_put_contents($vendorGeneratorPath, file_get_contents($patchGeneratorPath));
 }
+
+if (PHP_VERSION_ID < 70205) {
```

Comment:
Just wanted to see what this would do, but it still doesn't take care of the notice/stack trace from being printed into the test results.  We might want to go with the env var here